packages/vuexfire/types/index.d.ts

```diff
@@ -0,0 +1,11 @@
+import { MutationTree, ActionContext, Action } from 'vuex'
+import { firestore } from 'firebase'
+
+export declare const firebaseMutations: MutationTree<any>
+
+interface FirebaseActionContext<S, R> extends ActionContext<S, R> {
+  bindFirebaseRef(key: string, ref: firestore.DocumentReference | firestore.CollectionReference): Promise<void>
```

Comment:
We can give better hints for the promise here, as depending on the type of ref we get an array or a document (you can check `$bind` typings in vuefire package)

Reply:
Thanks.

I'll fix this.